oktapam/resource_resource_group.go

```diff
@@ -63,6 +64,11 @@ func resourceResourceGroupRead(ctx context.Context, d *schema.ResourceData, m an
 	if err != nil {
 		return diag.FromErr(err)
 	}
+	if resourceGroup == nil {
```

Comment:
nit Adding a comment here that client returns `nil` in case of 404 only will be better.

Reply:
Done